app/test3.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+def create_connection(db_url):
+    engine = create_engine(db_url)
+    base.metadata.bind = engine
+    dbsession = sessionmaker(bind=engine)
+    return dbsession()
```

Comment:
It's recommended to explicitly close the database session to free up resources and avoid potential memory leaks.

**Label**: best practice

**Code suggestion**

```python
session = dbsession()
try:
    # perform database operations
finally:
    session.close()
```